src/testdir/test_plugin_matchparen.vim

```diff
@@ -177,4 +177,28 @@ func Test_matchparen_ignore_sh_case()
   call StopVimInTerminal(buf)
 endfunc
 
+" Test for the WinScrolled event
+func Test_scroll_winenter()
```

Comment:
What does "winenter" have to do with this test?

Reply:
🤦 *win_something* ;)